### PR TITLE
7.4.0

### PR DESCRIPTION
**Pull request by robertohuertasm**

### CODE REVIEW

**Comment by JimiC** [comment]
Maybe we should remove the section in `README` about the icons conversion.

**Comment by robertohuertasm** [comment]
You're right. Let me know your thoughts 😉

File: README.md [hunk 1]
```diff
@@ -47,13 +47,16 @@ If you're willing to collaborate with us feel free to join our [Github repositor
 ---
 
 ## State of the extension
-**Release 6.0.0 introduced the ability for users to customize the icons without having to inject anything into the `Visual Studio Code`'s code.** 
 
-This was a huge step to allow the users to fully customize how the icons looked like and how they could be associated to any extension at will.
+With **release 7.4.0**, as a result of hearing you out, we've reverted how the `project detection` works. Now `Angular` icons are disabled by default. So if you want to enable them you will have to actively set the preset to true or rely on the `project detection` functionality.
```
Comment (JimiC) [inline README.md]: Consider changing `reverted` to `revised`.

File: README.md [hunk 2]
```diff
@@ -47,13 +47,16 @@ If you're willing to collaborate with us feel free to join our [Github repositor
...
+
+[@JimiC](https://github.com/JimiC) has improved and further optimized this functionality specifically for this release. But he also has provided the extension with the ability to support multi-language! Take a look at the translation's section if you want to know more or contribute.
 
-`Custom Icon associations` is still not supported by `VSCode API` but we thought that it was a very demanded feature and thought of a way to make it work for you. Remember that there's still an [open issue in VSCode's repo](https://github.com/Microsoft/vscode/issues/12493#issuecomment-249117649) talking about how they could provide official support for this feature. Provide a +1 if you want the VSCode team to support it out of the box. More information in [#328](https://github.com/vscode-icons/vscode-icons/issues/328).
+We still support the the ability for users to customize the icons without having to inject anything into the `Visual Studio Code`'s code.
```
Comment (robertohuertasm) [inline README.md]: Fixed!

File: README.md (hunk 1 above)
Comment (robertohuertasm) [inline README.md]: Done!

**Comment by JimiC** [comment]
LGTM.

**Comment by robertohuertasm** [comment]
I'm having issues with `nyc` while trying to publish. Once I use `vsce publish` I get an error:

```
npm ERR! Failed at the vscode-icons@7.4.0 test script 'nyc -c -x '' mocha ./out/test --recursive'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the vscode-icons package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     nyc -c -x '' mocha ./out/test --recursive
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs vscode-icons
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls vscode-icons
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\GIT\github\robertohuertasm\vscode-icons\npm-debug.log
```

After this, when I do `npm test` I get 3 errors:

```
1) i18n: tests ensure that each command title has an nls entry:
     AssertionError: expected undefined to exist
      at doAsserterAsyncAndAddThen (node_modules\chai-as-promised\lib\chai-as-promised.js:293:29)
      at Assertion.<anonymous> (node_modules\chai-as-promised\lib\chai-as-promised.js:283:21)
      at Assertion.get (node_modules\chai\lib\chai\utils\overwriteProperty.js:50:37)
      at packageJson.contributes.commands.forEach.command (out\test\i18n.test.js:2:6166)
      at Array.forEach (native)
      at Context.<anonymous> (out\test\i18n.test.js:2:5842)

  2) i18n: tests ensure that each configuration title has an nls entry:
     AssertionError: expected undefined to exist
      at doAsserterAsyncAndAddThen (node_modules\chai-as-promised\lib\chai-as-promised.js:293:29)
      at Assertion.<anonymous> (node_modules\chai-as-promised\lib\chai-as-promised.js:283:21)
      at Assertion.get (node_modules\chai\lib\chai\utils\overwriteProperty.js:50:37)
      at Context.<anonymous> (out\test\i18n.test.js:2:6747)

  3) i18n: tests ensure that each configuration description has an nls entry:
     AssertionError: expected undefined to exist
      at doAsserterAsyncAndAddThen (node_modules\chai-as-promised\lib\chai-as-promised.js:293:29)
      at Assertion.<anonymous> (node_modules\chai-as-promised\lib\chai-as-promised.js:283:21)
      at Assertion.get (node_modules\chai\lib\chai\utils\overwriteProperty.js:50:37)
      at Context.<anonymous> (out\test\i18n.test.js:2:7765)
```
I'm looking into it. For the moment, I've seen that in the `vscode:prepublish` script there's no need of executing `npm run precompile` as it is already triggered by the `build` script.

**Comment by robertohuertasm** [comment]
It seems that `vsce` has a bug. If you let the tool to upgrade your version automatically it will also apply the translations... For the moment I'm upgrading the version manually and gathering information about the bug in order to fix it.

**Comment by robertohuertasm** [comment]
Details of the bug:

- issue: [vscode-vsce#158](https://github.com/Microsoft/vscode-vsce/issues/158).
- PR: [vscode-vsce#159](https://github.com/Microsoft/vscode-vsce/pull/159)

**Pull request merged**